### PR TITLE
chore(root): update codeowners

### PR DESCRIPTION
**Pull request by bitgoAaron**
- make CODEOWNERS a little easier for everyone

Ticket: BG-00000

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -6,85 +6,81 @@
...
-
-# NOTHING BELOW THIS LINE
-CODEOWNERS @BitGo/git-admins @BitGo/sdk-team
+# All other coins
+modules/sdk-coin-aca/ @BitGo/coins
+modules/sdk-coin-ada/ @BitGo/coins
+modules/sdk-coin-algo/ @BitGo/coins
+modules/sdk-coin-avaxc/ @BitGo/coins
+modules/sdk-coin-avaxp/ @BitGo/coins
+modules/sdk-coin-bcha/ @BitGo/coins
```
Comment (tylerlevine) [inline CODEOWNERS:44]: this can be owned by the bitcoin team - it's a bitcoin cash fork

File: CODEOWNERS [hunk 2]
```diff
@@ -6,85 +6,81 @@
...
-modules/sdk-coin-btg/src/ @BitGo/btc-team
-modules/sdk-coin-btg/test/ @BitGo/btc-team
-modules/sdk-coin-dash/src/ @BitGo/btc-team
-modules/sdk-coin-dash/test/ @BitGo/btc-team
-modules/sdk-coin-ltc/src/ @BitGo/btc-team
-modules/sdk-coin-ltc/test/ @BitGo/btc-team
-modules/sdk-coin-zec/src/ @BitGo/btc-team
-modules/sdk-coin-zec/test/ @BitGo/btc-team
+modules/blockapis/ @BitGo/btc-team
+modules/bls-dkg/ @johnoliverdriscoll
```
Comment (tylerlevine) [inline CODEOWNERS:14]: we should avoid having individual accounts own modules - maybe use HSM team instead?

**Review by tylerlevine — CHANGES_REQUESTED** [review]
minor changes. We should probably also start a team for altcoins if we don't have one already

**Review by joshdk — APPROVED** [review]

**Review by tylerlevine — CHANGES_REQUESTED** [review]
Codeowners file contains errors, could you please take a look?

File: CODEOWNERS [hunk 3]
```diff
@@ -6,85 +6,81 @@
...
-modules/sdk-coin-btg/src/ @BitGo/btc-team
-modules/sdk-coin-btg/test/ @BitGo/btc-team
-modules/sdk-coin-dash/src/ @BitGo/btc-team
-modules/sdk-coin-dash/test/ @BitGo/btc-team
-modules/sdk-coin-ltc/src/ @BitGo/btc-team
-modules/sdk-coin-ltc/test/ @BitGo/btc-team
-modules/sdk-coin-zec/src/ @BitGo/btc-team
-modules/sdk-coin-zec/test/ @BitGo/btc-team
+modules/blockapis/ @BitGo/btc-team
+modules/bls-dkg/ @BitGo/hsm
```
Comment (TheRealDwright) [inline CODEOWNERS:14]: You'll need to provide the HSM Team write access to the SDK repo due to this change. Please reach out if you need help on how to do this.

Comment (mmcshinsky-bitgo) [reply CODEOWNERS:14]: Thanks. I've gone ahead and created the ticket and PR: https://bitgoinc.atlassian.net/browse/BG-56932

**Review by tylerlevine — APPROVED** [review]

**Pull request merged**